spec/js/views/add_todo_view_spec.js

```diff
@@ -8,9 +8,9 @@ define([
   var addToDoView;
 
   beforeEach(function () {
-    var toDosView = new ToDosView({model: new ToDoCollection(), el: $('body')});
+    var collection = new ToDoCollection();
 
-    addToDoView = new AddToDoView({model: toDosView, el: $('body')});
+    addToDoView = new AddToDoView({collection: collection, el: $('body')});
```

Comment:
Inconsistent naming, in other test suites you are using "instance" for an instance of the class under test.